index.html

```diff
@@ -50,7 +51,258 @@ <h3 class="header-highlight-2">I’am a Software Developer</h3>
             <a href="#"><i class="fab fa-github"></i></a>
           </li>
         </ul>
+        <img
+          class="hero-bg-img"
+          src="./images/background.png"
+          alt="hero backround image"
+        />
       </div>
-    </header>
+    </section>
+    <div class="grid">
+      <section class="recent-work-container">
+        <div class="recent-work-header">
+          <h2 class="recent-work-title">My Recent Works</h2>
+          <hr class="content-divider" />
+          <div></div>
+        </div>
+        <div class="card-placeholder-group">
+          <div class="featured-img">
+            <div class="img-placeholder"></div>
+            <div class="right-block">
+              <h2>Multi-Post Stories</h2>
+              <p class="supporting-text">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+
+              <ul class="skills-tags">
+                <li class="tags tag-1">
+                  <a id="tag-clr-1" href="#">css</a>
+                </li>
+                <li class="tags tag-2">
+                  <a id="tag-clr-2" href="#">html</a>
+                </li>
+                <li class="tags tag-3">
+                  <a id="tag-clr-3" href="#">bootstrap</a>
+                </li>
+                <li class="tags tag-4">
+                  <a id="tag-clr-4" href="#">Ruby</a>
+                </li>
+              </ul>
+              <button type="button" class="button btn-1">see project</button>
```

Comment:
- [ ] You did a great job adding the list of skills that you needed for the project but you don't need to wrap them with tag a because they are not links. Kindly fix it.